com.unity.multiplayer.mlapi/Runtime/Metrics/INetworkMetrics.cs

```diff
@@ -0,0 +1,51 @@
+using System.Collections.Generic;
+
+namespace MLAPI.Metrics
+{
+    public interface INetworkMetrics
```

Comment:
Maybe add a short XML comment that this interface is for us internally to connect to the tools. Because our users will see this interface in the API docs

Reply:
Made this and others internal since they're not meant to be exposed to the end-user.